utilities/naljal-custom-service/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM egovio/apline-jre:8u121
+
+MAINTAINER Senthil<senthil.kalimuthu@tarento.com>
```

Hadolint (2.12.0)

[error] MAINTAINER is deprecated

(DL4000)